### PR TITLE
Improve bl-user-setup

### PR DESCRIPTION
**Pull request by ghost**
Superseedes and incorporates PR #52 

 * Use HOME variable from lightdm too
 * For LDAP/AD users, GROUP might not be equal to user. Use GROUP passed
   by lightdm

@johnraff, is this easy to test?

### CODE REVIEW

File: bl-user-setup [hunk 1]
```diff
@@ -44,11 +58,11 @@ do
...
-ln -sTf /usr/share/images/bunsen/wallpapers "/home/$USER/Pictures/wallpapers/bunsen"
+ln -sTf /usr/share/images/bunsen/wallpapers "$HOME/Pictures/wallpapers/bunsen"
 
-mkdir -p "/home/$USER/.config/bunsen" # this should already exist
-touch "/home/$USER/.config/bunsen/bl-setup"
+mkdir -p "$HOME/.config/bunsen" # this should already exist
+touch "$HOME/.config/bunsen/bl-setup"
 
-chown -R "$USER":"$USER" "/home/$USER"
+chown -R "$USER":"$USERGROUP" "$HOME"
```
Comment (johnraff) [inline bl-user-setup]: "$USERGROUP"should be "$GROUP"

**Comment by johnraff** [comment]
> @johnraff, is this easy to test?

Easy to test at the local level: substitute the code, create a new user and check that the config is setup correctly. I can do that today. 
EDIT: Tested OK locally, with some small tweaks, as per https://github.com/BunsenLabs/bunsen-configs/pull/54

However, I have no knowledge of LDAP/AD logins, so will have to leave that part to someone else.

**Comment by johnraff** [comment]
@2ion is this OK to merge (before it's forgotten)?

**Comment by ghost** [comment]
Yes I think so. Thanks.

**Pull request merged**